.github/workflows/ci-it.yml

```diff
@@ -129,9 +129,9 @@ jobs:
       - name: Run SQL Lint
         run: make miglint
       - name: Run Go Lint
-        uses: golangci/golangci-lint-action@v2.5.2
+        uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.46
+          version: latest
```

Comment:
Check whether there are compatibility issues with `latest`.

Reply:
1.46 it's not compatibility with golang 1.17
maybe latest it's ok
if `latest` it's compatibility, `golang-lint` action will be ok